moveit_ros/move_group/src/default_capabilities/get_group_urdf_capability.cpp

```diff
@@ -56,7 +56,11 @@ const auto ROBOT_ELEMENT_CLOSING = std::string("</robot>");
 const auto GENERAL_ELEMENT_CLOSING = std::string("/>");
 }  // namespace
 
-GetUrdfService::GetUrdfService() : MoveGroupCapability("get_group_urdf")
+GetUrdfService::GetUrdfService()
+  : MoveGroupCapability("get_group_urdf")
+  , robot_description_subscriber_{ context_->moveit_cpp_->getNode()->create_subscription<std_msgs::msg::String>(
+        "robot_description", rclcpp::SystemDefaultsQoS(),
```

Comment:
I thought we discussed getting `robot_description` from the robot state publisher, so I expected this to be namespaced with that node. This reads as if its getting it from the `moveit_cpp` node. What am I missing here?

Reply:
I think it is just using the moveit_cpp node to create the subscription. The description as far as I can tell is published by the robot state publisher to a unamespaced topic: `/robot_description` unless you put the robot state publisher in a namespace which we don't right now.